README.md

```diff
@@ -157,11 +124,11 @@ make
 
 If you'd like to develop using the tools on your host and preserve the changes
 to source and revision control but still use the container for building, you
-can do using by mouning a volume in the container, using the `-v` option to mount
+can do using by mounting a volume in the container, using the `-v` option to mount
```

Comment:
No from your change, but as you already fixed something in the sentence, I suggest:
```suggestion
can do so by mounting a volume in the container, using the `-v` option to mount
```
... Or any better wording.